File: C#/SingleThreaded/webDownload.cs
using System;
using System.Diagnostics;
using System.Collections.Generic;
using System.Net;

class Program
{
  static WebClient client = new WebClient();

  static string getRequest(string url) {
        try
        {
            string html = client.DownloadString(url);
            return html;
        }
        catch (WebException e)
        {
            Console.WriteLine($"Error: {e.Message}");
            return e.Message;
        }
  }

  static void Main()
  {
    // Setup
    Stopwatch stopwatch = new Stopwatch();

    // Perform selection sorts and measure time
    stopwatch.Start();
    for (int i=0; i < 16; i++) {
      getRequest("https://esports.mst.edu");
    }
    stopwatch.Stop();
    TimeSpan duration = stopwatch.Elapsed;
    double durationSecs = duration.TotalMilliseconds/1000;
    string durationStr = durationSecs.ToString("0.000000");

    // Print Results
    Console.WriteLine(durationStr);
  }
}

File: Python/SingleThreaded/webDownload.py
import requests
import time

def getRequest(url):
    requests.get(url)

if __name__ == "__main__":
  # Download webpages and record time
  start = time.perf_counter()
  for i in range(0, 16):
    getRequest("https://google.com")
  stop = time.perf_counter()
  duration = stop - start

  # Print Results
  print(f'{duration:5f}')